XIVComboExpanded/Combos/BLM.cs

```diff
@@ -126,6 +126,8 @@ protected override uint Invoke(uint actionID, uint lastComboMove, float comboTim
                                     return BLM.Fire3;
                                 if (level > BLM.Levels.Paradox && gauge.IsParadoxActive)
                                     return BLM.Paradox;
+                                if (level > BLM.Levels.Despair && LocalPlayer?.CurrentMp > 0 && (HasEffect(BLM.Buffs.Swiftcast) || HasEffect(BLM.Buffs.Triplecast)))
```

Comment:
The `HasEffect` calls here would seem to be redundant with the conditional (line 123) that this is already within, no?

Reply:
![image](https://github.com/user-attachments/assets/bd9a198c-59f8-49fe-bf87-6beed9931811)
No because there is a route where the gcd before was is a hardcast or a xeno.

Reply:
Fixed in second commit, there might be some edge cases but the opener sequence seems ok.

Reply:
oh wait.  No, I read the conditional wrong.  If the last cast, after the `II`, is the case, you won't have swift or triple.  I think I was wrong on this review comment, I think you might actually need these checks, or it may try to hardcast Despair.

Reply:
Sure, will revert.